specifyweb/context/views.py

```diff
@@ -823,6 +823,14 @@ def languages(request):
         else:
             languages = languages.split(',')
 
+        temp = JsonResponse({
+            code:{
+                **get_language_info(code),
+                'is_current': code==request.LANGUAGE_CODE
+            } for code in languages
+        })
+        print(temp)
```

Comment:
Is this `temp` variable used anywhere else? Can it be removed?

Reply:
good catch, removed that temp code